### PR TITLE
sbt test patch + exposed jes bug fix 

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala [hunk 1]
```diff
@@ -495,16 +495,19 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
...
-      case (Seq(eventCurrent, eventNext), index) =>
-        val eventKey = s"executionEvents[$index]"
-        tellMetadata(s"$eventKey:description", eventCurrent.name)
-        tellTimeMetadata(s"$eventKey:startTime", eventCurrent.timestamp)
-        tellTimeMetadata(s"$eventKey:endTime", eventNext.timestamp)
+    eventList.headOption foreach { firstEvent =>
+      // The final event is only used as the book-end for the final pairing so the name is never actually used...
+      val offset = firstEvent.timestamp.getOffset
+      val now = OffsetDateTime.now.withOffsetSameInstant(offset)
+      val lastEvent = EventStartTime("unused_name", now)
```
Comment (cjllanwarne) [inline supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala]: I just saw that the Queen's jetpacking has been removed in Cromwell v2. Much sadness.

**Comment by cjllanwarne** [comment]
👍 assuming a Travis is happy

File: supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala (hunk 1 above)
Comment (Horneth) [inline supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala]: If I know one thing it's that you can't get rid of the Queen that easily, she'll be back.

**Comment by ruchim** [comment]
Tried, tested and true 👍

**Comment by kshakir** [comment]
Unfortunately Travis isn't as fast as our local machines. 😢 

Travis was timing out on the hard stop 3 second limit imposed by default in Akka TestKit. This wasn't an issue until _this_ PR started running those tests on the slower Travis VMs. I've dilated the time travis takes to run in the latest commit to account for travis being slower than our machines.

Will also log a ticket regarding the extensive [actor system leaks](https://github.com/akka/akka/blob/v2.3.12/akka-testkit/src/main/scala/akka/testkit/TestKit.scala#L705-L706).

**Comment by kshakir** [comment]
Dilation failed, as the affected tests seemed to ignore the default config. I ~~hacked~~ put a bandaid solution in for now and left `PBE` notes todo later.

**Comment by mcovarr** [comment]
I concur with your TODO on DRYing up the Akka configs, but still :+1:

**Comment by scottfrazer** [comment]
👍

**Comment by kshakir** [comment]
Cool. Squashing a little, then merging.

**Pull request merged**